lua/wire/client/text_editor/wire_expression2_editor.lua

```diff
@@ -1818,13 +1818,9 @@ function Editor:SaveFile(Line, close, SaveAs)
 		return
 	end
 
-	local path = ""
-	for k,v in ipairs(string.Explode("/", Line)) do
-		if string.EndsWith(v, ".txt") then continue end
-		path = path..v.."/"
-		if not file.IsDir(path, "DATA") then
-			file.CreateDir(path)
-		end
+	local path = string.Left(Line, -(#string.GetFileFromFilename(Line)+1))
```

Comment:
Can't this just be `path = string.GetFileFromFilename(Line)`?

Reply:
No, but it can be  path = string.GetPathFromFilename(Line). Another function I didn't know existed.